### ISSUE TITLE
Bug with v8 Paths, but not sure what is the root cause

### ISSUE THREAD

**Issue by Aghassi** [cenfun/jest-monocart-coverage#5]
I followed the example provided in the README, but the issue I'm seeing with my system is that the v8 data strips out the sourcePath from the file. This does not happen if the jest config does not have `coverageProvider: 'v8'`. The thing is, I'm not sure _why_ this is happening. I'm opening this issue more to discuss. I have not made a way to publicly reproduce because this is on our internal monorepo with Bazel.

Basically, the lcov file generate is missing `path/to/file.ts` with that option on. It just returns `file.ts`.

I have some theories, but nothing concrete. Any ideas?

**Comment by cenfun** [comment]
This path issue could be a sourcemap issue. The sourcemap of the file didn't provide the full path of the file.
```js
{"version":3,"sources":["file.js"],"sourcesContent":[ ...
```
It is supposed to be:
```js
{"version":3,"sources":["path/to/file.js"],"sourcesContent":[ ...
```
Please try to normalize the `sourcePath` with option `sourcePath`.
Possible solutions:
```js
const path = require("path")

// MCR coverage options
{
  sourcePath: (filePath, info)=> {
    if (!filePath.includes('/') && info.distFile) {
      return `${path.dirname(info.distFile)}/${filePath}`;
    }
    return filePath;
  },
  
}
```

**Comment by Aghassi** [comment]
Thanks I believe that did it! That's definitely worth documenting in the README. Should it go in this repo or the monocart main one?

**Comment by cenfun** [comment]
Thank you for the reminder, I have updated the document for both `jest-monocart-coverage` and `monocart-coverage-reports` repo.
